src/data/history.ts

```diff
@@ -55,7 +55,8 @@ export const fetchRecent = (
   entityId,
   startTime,
   endTime,
-  skipInitialState = false
+  skipInitialState = false,
+  includeLocation = false
```

Comment:
Pretty sure that this is not how we're going to call these options.

Probably be more likely to be called `fillStartPeriod` and `significantChangesOnly`


CC @Santobert

Reply:
@balloob This is the backend pr: https://github.com/home-assistant/core/pull/33356

I've renamed `includeLocation` to `significantChangesOnly`. `skipInitialState` was already there and was not introduces by us, so I've decided to leave is as it is for now.